integration_tests/src/main/python/array_test.py

```diff
@@ -103,11 +103,13 @@
 
 @pytest.mark.parametrize('data_gen', array_item_test_gens, ids=idfn)
 @pytest.mark.parametrize('index_gen', array_index_gens, ids=idfn)
+@disable_ansi_mode
```

Comment:
Is there a followup issue to remove this marker and fix this properly by explicitly setting the ANSI modes for tests?  This should not be a long-term fix for ANSI-related testing.

Reply:
[Yes](https://github.com/NVIDIA/spark-rapids/issues/11106), the plan is to revisit the tests and re-write tests and possibly incorporate the changes we discussed offline